exercises/twelve-days/src/test/java/TwelveDaysTest.java

```diff
@@ -119,14 +119,26 @@ public void testVerseTwelve() {
 
     @Ignore("Remove to run test")
     @Test
-    public void testMultipleVerses() {
-        String expectedVerseOneToThree = "On the first day of Christmas my true love gave to me, " +
+    public void testFirstThreeVerses() {
+        String expectedVersesOneToThree = "On the first day of Christmas my true love gave to me, " +
                 "a Partridge in a Pear Tree.\n\n" +
                 "On the second day of Christmas my true love gave to me, two Turtle Doves, " +
                 "and a Partridge in a Pear Tree.\n\n" +
                 "On the third day of Christmas my true love gave to me, three French Hens, two Turtle Doves, " +
                 "and a Partridge in a Pear Tree.\n";
-        assertEquals(expectedVerseOneToThree, twelveDays.verses(1, 3));
+        assertEquals(expectedVersesOneToThree, twelveDays.verses(1, 3));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testFourthToSixthVerses() {
+        String expectedVersesFourToSix = "On the fourth day of Chrsitmas my true love gave to me, " + 
+                "four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n\n" +
+                "On the fifth day of Christmas my true love gave to me, five Gold Rings, four Calling Birds, " + 
+                "three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n\n" +
+                "On the sixth day of Christmas my true love gave to me, six Geese-a-Laying, five Gold Rings, " + 
+                "four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n"
```

Comment:
Did a quick look over this. @Kyle-P you are missing a ; at the end of line 140. Hide and seek champion of the world. :)

Reply:
Thanks @c-thornton! Nice catch!